src/lib/pickles/plonk_checks/plonk_checks.ml

```diff
@@ -159,25 +158,23 @@ let expand_feature_flags (type boolean)
          ] )
   in
   let lookups_per_row_4 =
-    (* Xor, ChaChaFinal, RangeCheckGate have max_lookups_per_row = 4 *)
+    (* Xor, RangeCheckGate, ForeignFieldMul, have max_lookups_per_row = 4 *)
     lazy
-      (B.( ||| )
-         (Lazy.force lookup_pattern_xor)
-         (Lazy.force lookup_pattern_range_check) )
+      (B.any
+         [ Lazy.force lookup_pattern_xor
+         ; Lazy.force lookup_pattern_range_check
+         ; foreign_field_mul
```

Comment:
`Lazy.force` required here?

Reply:
Make the change here and then merge this branch into your development one to get the additional change?

Reply:
I would try that, hopefully that won't break the commit history 👀

Reply:
If you look at the modified code, the foreign-field-mul was never a lazy. Actually the type does not match and cannot do it with Lazy.force. Thus, I assume it is correct the way it is now, or at least it is not modifying what we already had before.

Reply:
Because foreign_field_mul is just a bool